src/server/replica.h

```diff
@@ -145,6 +146,16 @@ class Replica : ProtocolClient {
   // Get the current replication phase based on state_mask_
   std::string GetCurrentPhase() const;
 
+  // Used *only* in TakeOver flow. There is small data race if
+  // thread_flow_map_ gets written by the MainReplicationFiber thread but
+  // the chances for that are extremely rare.
+  std::vector<unsigned> GetFlowMapAtIndex(size_t index) const {
+    DCHECK(index < thread_flow_map_.size());
+    return thread_flow_map_[index];
```

Comment:
thread_flow_map_.at(index) is checked in debug

Reply:
Isn't it the case that `.at()` performs a run time check regardless of build type ? It's not only in debug basically